content/education/manage-coursework-with-github-classroom/teach-with-github-classroom/about-assignments.md

```diff
@@ -0,0 +1,52 @@
+---
+title: About assignments
+intro: 'You can use individual or group assignments in {% data variables.product.prodname_classroom %} to teach students and grade their work.'
+shortTitle: About assignments
+type: overview
+topics: 
+  - Fundamentals
+versions:
+  fpt: '*'
+---
+
+## About assignments in {% data variables.product.prodname_classroom %}
+
+Assignments are coursework created for students in {% data variables.product.prodname_classroom %}. You can use assignments to test and grade your students, or to help your students practice their learnings.
+
+With {% data variables.product.prodname_classroom %}, you can create individual assignments, to be completed by a single student, or group assignments, to be completed in teams. For more information, see "[Types of assignments](#types-of-assignments)."
+
+{% data reusables.classroom.about-assignments %}
+
+{% data reusables.classroom.about-feedback-pull-requests %}
+
+{% data reusables.classroom.about-ide-integration %}
+
+{% data reusables.classroom.reuse-assignment-link %}
+
+{% data reusables.classroom.assignments-classroom-prerequisite %}
```

Comment:
```suggestion
{% note %}

**Note:** {% data reusables.classroom.assignments-classroom-prerequisite %}

{% endnote %}
```

Since this section is pretty long, I'd make this a note so it stands out from the rest of the text as particularly important information!